### PR TITLE
feat(healthie): Add checkPatientTag and checkScheduledAppointments actions

### PR DESCRIPTION
**Pull request by danijel**
- Add checkPatientTag healthie action that checks does patient has an active tag with id
- Add checkScheduledAppointments action that check does the patient have active future appointment with appointment_type_id

### CODE REVIEW

**Comment by danijel** [comment]
@bejoinka, @ebomcke-awell  tests a failing because the addition of the external-server, I think there is some issues with the naming of this extension and it's action.

I see this was added recently but I don't know what is the purpose of this, is this even used, can it be disabled. We need these two healthie actions for bootcamp with a customer

File: extensions/healthie/actions/checkPatientTag/checkPatientTag.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+export const checkPatientTag: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'checkPatientTag',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Check patient tag',
+  description: 'Check is tag applied as active tag on patient in Healthie.',
```
Comment (ebomcke-awell) [inline extensions/healthie/actions/checkPatientTag/checkPatientTag.ts:17]:
```suggestion
  description: 'Check if a patient has the given tag in their active tags in Healthie.',
```

File: extensions/healthie/actions/checkScheduledAppointments/checkScheduledAppointments.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+export const checkScheduledAppointments: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'checkScheduledAppointments',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Check scheduled appointments',
+  description:
+    'Check does the patient have active appointment scheduled for an appointment type',
```
Comment (ebomcke-awell) [inline extensions/healthie/actions/checkScheduledAppointments/checkScheduledAppointments.ts:18]:
```suggestion
    'Check if the given patient has an active appointment scheduled for the given appointment type.',
```

**Review by ebomcke-awell — CHANGES_REQUESTED** [review]
I find the pattern used for test pretty confusing. Reading the tests themselves it's not immediately apparent why the result of `onComplete` changes in the different test scenarios. It's only when looking at the mock SDK file that it becomes clear: it uses the input payload to conditionally generate API responses.
This is not great as it makes tests less understandable and harder to maintain because of the semi-hidden dependency to the mock SDK logic.

Defining the mock implementation inside the test file would make this much clearer. Not necessarily needed for this PR though, I'm mentioning this for future reference.

The description changes on the other hand are required as they show up in Studio.

**Pull request merged**